<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TasklistEditor</class>
 <widget class="QWidget" name="TasklistEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>1097</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>必填参数</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>无</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>可选参数</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="checkBox_remote_system">
        <property name="text">
         <string>连接到远程系统</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_remote_system">
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_title_option_s">
             <property name="minimumSize">
              <size>
               <width>230</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>230</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>/S</string>
             </property>
             <property name="text">
              <string>主机地址/名称：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_option_s">
             <property name="toolTip">
              <string>/S</string>
             </property>
             <property name="placeholderText">
              <string>ip/url/主机名称</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_title_option_p">
             <property name="minimumSize">
              <size>
               <width>230</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>230</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>密码：</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEdit_option_p">
             <property name="placeholderText">
              <string>可选</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_option_u">
             <property name="toolTip">
              <string>/U</string>
             </property>
             <property name="placeholderText">
              <string>可选</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_title_option_u">
             <property name="minimumSize">
              <size>
               <width>230</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>230</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>/U</string>
             </property>
             <property name="text">
              <string>用户名：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_info_option_s">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_option_group2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_option_m">
             <property name="placeholderText">
              <string>dll模块名，如Shell32.dll，可不填</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBox_option_m">
             <property name="minimumSize">
              <size>
               <width>230</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>230</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>列出调用指定DLL的所有进程:</string>
             </property>
             <property name="autoExclusive">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBox_option_svc">
             <property name="minimumSize">
              <size>
               <width>230</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>230</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>显示每个进程中的服务</string>
             </property>
             <property name="autoExclusive">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBox_option_v">
             <property name="minimumSize">
              <size>
               <width>230</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>230</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>显示详细信息</string>
             </property>
             <property name="autoExclusive">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_option_fi">
        <property name="minimumSize">
         <size>
          <width>230</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>230</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>设置筛选器</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_filter">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item row="1" column="1">
            <widget class="QWidget" name="widget_operators" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QComboBox" name="comboBox_op_status"/>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_op_imageName"/>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_op_pid"/>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_op_session"/>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_op_sessionName"/>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_op_cpuTime"/>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_op_memusage"/>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_op_userName"/>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_op_service"/>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_op_windowTitle"/>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_op_module"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboBox_filterType"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>230</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>230</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>判断符：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>230</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>230</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>判断内容：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>230</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>230</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>选择筛选器：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QWidget" name="widget_filterValues" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QComboBox" name="comboBox_fiValue_status">
                <item>
                 <property name="text">
                  <string>RUNNING</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>SUSPENDED</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>NOT RESPONDING</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>UNKNOWN</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_fiValue_imageName">
                <property name="placeholderText">
                 <string>映像名称</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_fiValue_pid">
                <property name="placeholderText">
                 <string>PID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_fiValue_session">
                <property name="placeholderText">
                 <string>会话编号</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_fiValue_sessionName">
                <property name="placeholderText">
                 <string>会话名称</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTimeEdit" name="timeEdit_fiValue_cpuTime">
                <property name="showGroupSeparator" stdset="0">
                 <bool>false</bool>
                </property>
                <property name="maximumTime">
                 <time>
                  <hour>23</hour>
                  <minute>59</minute>
                  <second>59</second>
                 </time>
                </property>
                <property name="displayFormat">
                 <string>HH:mm:ss</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_fiValue_memUsage">
                <property name="suffix">
                 <string>KB</string>
                </property>
                <property name="maximum">
                 <number>1000000000</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_fiValue_userName">
                <property name="placeholderText">
                 <string>用户名，格式为：[域\]用户</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_fiValue_service">
                <property name="placeholderText">
                 <string>服务名称</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_fiValue_windowTitle">
                <property name="placeholderText">
                 <string>窗口标题</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_fiValue_module">
                <property name="placeholderText">
                 <string>模块名称</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_option_nh">
          <property name="minimumSize">
           <size>
            <width>230</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>230</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>不显示表头</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox_option_fo">
          <property name="minimumSize">
           <size>
            <width>230</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>230</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>指定输出格式</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox_option_fo">
          <item>
           <property name="text">
            <string>TABLE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LIST</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CSV</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
